### DISCUSSION TITLE
Oz Smart Things RGB Downlight - turns off immediately after turning on with HA

### DISCUSSION THREAD

**Discussion by Gurgeh88** [Koenkk/zigbee2mqtt#17735] [Devices]
Posted this somewhere else but not sure if it was the right spot so here we go again. Hopefully someone can point me in the right direction:

Have recently installed two of these lights and integrated into Home Assistant using Zigbee2MQTT.
Problem is, when I turn them on after they've been off for a little while, they turn off again straight away (instantly almost).
When this happens, they appear in HA UI as being on but they are definitely off.
The only way to get them to turn when this happens is to call the light to turn on again via HA.

Does anyone have any experience of this happening for them/have any suggestions for how to fix this?

My Home Assistant and Zigbee2MQTT are all up to date. This behaviour is completely isolated to these two new lights. Both of them have great link quality according to Zigbee2MQTT and don't show signs of regular service calls being unreliable. Only when they've been off for a while. I have reset them and re-paired with Zigbee2MQTT which didn't help. Have also tried adding them to the Tuya Smart or Smart Life apps on my phone but was not able to do this successfully (couldn't work out which option to pick from the Add Device list and nothing showed up when scanning for devices ready to pair). I wanted to do this so I could update the firmware.

Any help would be greatly appreciated as otherwise I like these bulbs and would happily use them in the rest of the house but for this one flaw which makes them pretty much useless.

Since originally encountering this problem, I've tried putting the lights in a Zigbee2MQTT group rather than a Home Assistant UI group and that has not changed the behaviour.

**Comment by Gurgeh88** [comment]
Can anyone offer some advice on how to further debug this issue? Still remains despite latest Z2M update.

**Comment by Gurgeh88** [comment]
Just an update, I purchased a Tuya Zigbee hub to see if there was an update available (as Z2M showed none).
No luck there.
These lights are on the latest firmware according to the Tuya app.
Unfortunately, without any extra support to fix these lights I couldn't honestly say they were worth buying and using with Z2M and Home Assistant. My bodge job automation to get them to work is simply not clean enough.

**Reply by zuohaot** [reply]
I have the same problem mate. I will write an email to ozsmartthings. Please don't mind me quoting your post here.

**Reply by zuohaot** [reply]
I was wondering if these lights behave in the same way when you connect them to your Tuya Zigbee hub. If not, I think we can simply use Zemismart Homekit hub to port them to HA.

**Reply by Gurgeh88** [reply]
Thanks for posting. They seemed to work fine when connected the Tuya hub. Would it be problematic to have another ZigBee controller operating on my network though?

**Reply by Gurgeh88** [reply]
I didn't try porting them into HA using the Tuya hub though, just used it to look for a firmware update. I have the Tuya integration already for a grid connect heater so I could try adding them in with that if I went down that road.

**Reply by zuohaot** [reply]
You should be fine. I assume zigbee gateway from a propritary manufacturer, such as Tuya and Aqara, should be able to change their channels when there is interference?? I am using Z2M with Aqara M2 zigbee gateway right next to it without any issue. I will defintely try to run a Tuya gateway at the same time to see how it goes.
Also, I got a response from Oz Smart Things as well, they listed many possible factors but none of them is really related to what we care about, except the firmware update.
